src/main/java/org/mule/service/http/impl/service/util/RedirectUtils.java

```diff
@@ -95,10 +97,14 @@ public HttpRequest createRedirectRequest(HttpResponse response, HttpRequest requ
     headers.remove(Host.toString());
     headers.remove(ContentLength.toString());
     String redirectMethod;
+    HttpEntity entity = request.getEntity();
 
     if (mustBeSendAsGet(response.getStatusCode())) {
       redirectMethod = GET.name();
       headers.remove(ContentType.toString());
```

Comment:
Have you checked the case-sensivity thing mentioned in the ticket?

Reply:
yes, it does not affect this issue